src/index.css

```
@@ -5,13 +5,31 @@
 @tailwind utilities;
 
 @layer base {
+  html {
+    background-color: var(--paper);
+  }
+
   body {
     @apply min-h-screen m-0;
     background-color: var(--paper);
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)